lighthouse-core/report/templates/report-template.html

```diff
@@ -153,9 +153,9 @@ <h2 class="config-section__title">Blocked URL Patterns</h2>
                 <li class="subitem {{#if subItem.comingSoon}}--coming-soon{{/if}} {{#if (shouldShowHelpText subItem.score)}}--show-help{{/if}}">
 
                   <p class="subitem__desc">
-                    {{#unless ../../scored }}
+                    <!--{{#unless ../../scored }}
```

Comment:
I left this in for now in case we want to bring it back some day. Also left the category in each audit.